dev/scripts/src/alluxio.org/check-docs/main.go

```diff
@@ -328,7 +328,10 @@ func checkAndSaveUrl(files []File, menuMap map[string]struct{}, errMsgs []string
 
 // Check menu.yml URLs should match exactly with list of all markdown doc files
 func checkUrlMatch(docsPath, checkPath string, menuListOfURL map[string]struct{}) error {
+	// map for  all path of markdown file that actually in the directory of check path.
 	fileList := make(map[string]struct{})
+	// map for  all name of markdown file that actually in the directory of check path.
+	fileNameMap := make(map[string]string)
```

Comment:
```suggestion
	// set of markdown file paths
	filePaths := make(map[string]struct{})
	// map of markdown file name -> file path, to detect duplicate file names
	fileNameMap := make(map[string]string)
```
usually i recommend to describe the map key and value if its not obvious from the types (there are so many map[string]string types)